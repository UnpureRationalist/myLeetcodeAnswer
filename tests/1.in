27
7 2 14 18 4 6 20 18 13 11 1 14 15 14 18 22 24 25 9 7 9 19 11 2 18 3 18