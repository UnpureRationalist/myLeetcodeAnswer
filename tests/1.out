9 11 14 18